### ISSUE TITLE
Clearer documentation of what this plugin provides or does better than Transistorsoft

### ISSUE THREAD

**Issue by CodeWithOz** [HaylLtd/cordova-plugin-background-geolocation#31]
**Is your feature request related to a problem? Please describe.**
Nice work on this plugin! I'd like to get a better idea of what developers stand to gain by using this plugin instead of the one by Transistorsoft. The README only has this line:
> It is more battery and data efficient than html5 geolocation or cordova-geolocation plugin...

Transistorsoft claim that their plugin is the best one available, so I'd like to know some more about what this fork has improved upon.

**Describe the solution you'd like**
A discussion in this thread, and ultimately an update to the README of this plugin describing the improvements.

**Comment by HarelM** [comment]
My two cents:
I need to be able to record geo location while the app is not in the front. This can't happen using only html5 (long awaited request in w3c). This is a core requirement for my navigation app.
So you need native capabilities.
Transistor plugin is great, and I was willing to pay for it if delivered my needs (and I did and was refunded as it didn't delivered).
Unfortunately, the philosophy around it is battery optimization and I needed accuracy which are opposite philosophies...
Thus I switch to use this plugin which was better for my needs and also open for community changes and free, which is a bonus.
Hope this answers your question.
I'm not sure how to incorporate my experience in the readme. If you find a good way to write this down feel free to submit a PR :-)

**Comment by CodeWithOz** [comment]
@HarelM thanks for the quick response! Can you shed some light on the level of accuracy that was not sufficient for you? I haven't tried this plugin yet but from the docs it looks like the `desiredAccuracy` and `distanceFilter` config options have been preserved from the original Transistorsoft config options.

If it's not very easy to describe your experience, maybe try answering these questions:
- when was the location not accurate enough using Transistorsoft? Always, sometimes, rarely?
- was the accuracy of Transistorsoft inconsistent? For instance did the accuracy drop unexpectedly and unpredictably?
- do you have any measurements of how much more battery consumption this plugin uses compared to Transistorsoft? If the battery optimizations are reduced in order to improve accuracy, I would imagine that you experienced some kind of drop-off in battery performance when you switched to this plugin.

Thanks!

**Comment by HarelM** [comment]
Yeah, reading my comment back I wasn't very clear.
I'm not talking about GPS accuracy in terms if the location position but rather when is the position taken.
See:
https://github.com/transistorsoft/cordova-background-geolocation-lt/issues/980
Battery optimization philosophy dictates that the position should not be used if it's not needed, and I deemed it as bad user experience, my users expect recording to have all relevant points and not miss location because of plugin battery optimizations...
I hope this explains a bit my point of view...

**Comment by HarelM** [comment]
Also worth fixing along the way the provider broken link here:
![image](https://user-images.githubusercontent.com/3269297/123385909-c1542b80-d59e-11eb-9b47-899d1a4fa2c4.png)
First paragraph ("see PROVIDERS"):
https://haylltd.github.io/cordova-background-geolocation-plugin/api.html

**Comment by ArgelMichinel** [comment]
Greeting!
I wrote because I have around four months working to do something like a GPS app. I can go forward, however I would like that the app report to the server when it's in the background.
I began to use the Transistorsoft's plugin, but I can't reach that. In the begining I was thound that was because I don't was use of correct way the plugin, but then I realized that this part of the plugin doesn't work. The app don't reach to sent the location to the server with function autoSync active.
I want to know if with this plugin I can do that? So I don't spend time customizing the app to obtain negative results.

Thanks.
